README.md

```diff
@@ -36,19 +53,18 @@ git_repository(
     commit = "{HEAD}",
 )
 
-# Only needed for PIP support:
-load("@rules_python//python:pip.bzl", "pip_repositories")
+load("@rules_python//python:repositories.bzl", "py_repositories")
```

Comment:
Add a similar comment that this is needed to use py_library, etc.

Reply:
Clarified. I don't think it's important to say it's for the core rules specifically since all uses of this repo should also include this call.